client/package.json

```diff
@@ -161,7 +161,7 @@
     "stylelint-css-modules": "^0.9.0",
     "ts-jest": "^26.2.0",
     "ts-loader": "^9.4.3",
```

Comment:
But @lilitkarapetyan - shouldn't we try to use the most recent version of TS?

Reply:
Hmm, from the console it does look like we're running a more recent version:
![image](https://github.com/serge-web/serge/assets/1108513/f1bd9800-0afe-48b0-9c95-de387f3c5a53)